StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp12/DocumentationRules/SA1611CSharp12UnitTests.cs

```diff
@@ -3,9 +3,109 @@
 
 namespace StyleCop.Analyzers.Test.CSharp12.DocumentationRules
 {
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.Testing;
     using StyleCop.Analyzers.Test.CSharp11.DocumentationRules;
+    using Xunit;
+    using static StyleCop.Analyzers.Test.Verifiers.CustomDiagnosticVerifier<StyleCop.Analyzers.DocumentationRules.SA1611ElementParametersMustBeDocumented>;
 
     public partial class SA1611CSharp12UnitTests : SA1611CSharp11UnitTests
     {
+        [Fact]
```

Comment:
The tests for class and struct seem to be identical except the keyword, which is natural. You can declare a Theory instead of a Fact and cover both class and struct with less test code.